### ISSUE TITLE
CTD with possible mod incompatibility

### ISSUE THREAD

**Issue by Bernasss12** [Hirtol/PartyScreenEnhancements#3]
Hi, I've experienced this crash a couple of times today and since the dump seems to reference 2 mods I will report it to both creators.

### Here are the crash logs:
[Log1](https://gist.github.com/Bernasss12/72123fa4f5e25fbf6320d59835e67393)
[Log2](https://gist.github.com/Bernasss12/ec72e9c594861670a6bcdb6f487210a9)

### Versions:
Bannerlord: e1.2.1
PartyScreenEnhancements: 2.4.0
Improved Garrisons: 2.1.0 (Vortex didn't recognize the new version since the file name is different, I will try to see if the bug happens with version 3.0.0)

### Situation and repeatability:
I can't seem to make it happen 100% of the times but it only happened when fighting lords, right after all the dialogs, when loading the prisoner screen.

If you need any more details please do ask

**Comment by Hirtol** [comment]
Yeah, this is currently an issue on Improved Garrisons (2.1)

Improved Garrisons replaces the GarrisonpartySizeLimitModel which is used to query the total party size for displaying the shortened label.

That itself isn't anything to worry about usually, but it appears that they neglect some null checks somewhere, causing crashes as you have experienced.

According to another mod dev this issue has been fixed in Improved Garrisons (3.0), so I'd advise you download and try that. I'll keep the issue open for now until I've been able to confirm it's been fixed.

Thanks for the report and crash logs!
